ampbench_cli

```diff
@@ -54,12 +40,37 @@ const url = require('url');
 const util = require('util');
 const inspect_obj = (obj) => {return util.inspect(obj, { showHidden: true, depth: null })};
 const S = require('string');
-
 const handlebars = require('handlebars');
 
 var program = require('commander');
 
+//Nagios integration
+var criticalThreshold =  2;
+var warningThreshold =  1;
+var okThreshold = 0;
+
```

Comment:
I think it would not be a problem to have a const. I used var because I used the same bone that was in place on the old ampChecker.